torchvision/transforms/functional_tensor.py

```diff
@@ -757,7 +757,7 @@ def pad(img: Tensor, padding: List[int], fill: int = 0, padding_mode: str = "con
     return img
 
 
-def resize(img: Tensor, size: List[int], interpolation: int = 2) -> Tensor:
+def resize(img: Tensor, size: List[int], interpolation: str = "bilinear") -> Tensor:
```

Comment:
Again here, why we use the string value instead of its equivalent enum?

Reply:
Because, such that we should not map it to the arguments for torch functions.